src/jira/client/jira-client-issue-key-helper.test.ts

```diff
@@ -0,0 +1,667 @@
+import {
```

Comment:
These are all fresh tests, has super high coverage....
I have broken each function into its own describe block..... since the are all isolated its pretty easy to follow i think. I will have a re-review over these as well.